app/javascript/controllers/collapse_controller.js

```diff
@@ -0,0 +1,29 @@
+import { Controller } from '@hotwired/stimulus'
+import { Collapse } from 'bootstrap'
+
+// Connects to data-controller="collapse"
+export default class extends Controller {
+  static targets = ['collapsible']
+
+  initialize () {
+    const isExpanded = window.sessionStorage.getItem('filtersCollapsed') === 'false'
```

Comment:
This should incorporate a key that is settable in the view template, so it can be used on other pages in the app.